src/components/Form/Field.js

```diff
@@ -4,6 +4,7 @@ import {css, merge, simulate} from 'glamor'
 import colors from '../../theme/colors'
 import {fontFamilies} from '../../theme/fonts'
 import {mUp} from '../../theme/mediaQueries'
+import { pxToRem } from '../Typography/utils'
```

Comment:
is converting the field necessary? For the discussion preferences overlay role input?